modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/handle/migrate/MigrateImportDataProcess.java

```diff
@@ -26,7 +26,7 @@ public class MigrateImportDataProcess implements Serializable, JavaDelegate {
 
     @Autowired
     public void setRuntimeService(RuntimeService runtimeService) {
-        this.runtimeService = runtimeService;
+        MigrateImportDataProcess.runtimeService = runtimeService;
```

Comment:
please make these fields as non-static and autowire via constructor

Reply:
Same as above